core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.directive.html

```diff
@@ -51,8 +51,10 @@
           <span ng-if="!topic.is_published"> Not Published </span>
         </a>
       </td>
-      <td ng-if="canDeleteTopic()" class="delete-icon">
-        <span ng-click="deleteTopic(topic.id)" class="fas fa-trash-alt list-summary protractor-test-delete-topic-button"> </span>
+      <td class="delete-icon">
```

Comment:
Do we need the `delete-icon` class here? (if yes, then maybe we can move it to the anchor tag?)

Reply:
Removed the class, thanks!